solidity-bindgen/src/context.rs

```diff
@@ -0,0 +1,36 @@
+use std::sync::Arc;
+use web3::transports::EventLoopHandle;
+use web3::types::Address;
+
+/// A type needed to instantiate contracts. All contracts instantiated with the
+/// same Context will share the same EventLoopHandle. As long as the contracts
+/// are not dropped, the EventLoopHandle will not be dropped either. This type
+/// is cheap to clone.
+#[derive(Clone)]
+pub struct Context(Arc<ContextInner>);
+
+struct ContextInner {
+    url: String,
+    handle: EventLoopHandle,
+    from: Address,
+}
+
+impl Context {
+    pub fn new(url: String, from: Address) -> Result<Self, web3::error::Error> {
+        let handle = EventLoopHandle::spawn(|_| Ok(()))?.0;
```

Comment:
It's not clear to me why the `.0` here from looking at the EventLoopHandle struct in the docs.  Is the handle wrapped?